source/developers/projects/studio/api/repo/add-remote.rst

```diff
@@ -0,0 +1,130 @@
+|| remote_username     || String     || |checkmark| * || Username to use to access remote repository                 |
++----------------------+-------------+----------------+--------------------------------------------------------------+
+|| remote_password     || String     || |checkmark| * || Password to use to access remote repository                 |
++----------------------+-------------+----------------+--------------------------------------------------------------+
+|| remote_token        || String     || |checkmark| * || Token to use for authentication to access remote repository |
++----------------------+-------------+----------------+--------------------------------------------------------------+
+|| remote_auth_key     || String     || |checkmark| * || Authentication key to access remote repository              |
++----------------------+-------------+----------------+--------------------------------------------------------------+
+
+.. note::
+    ``*`` Required parameters:
+        * ``remote_username`` is required if ``autehentication_type`` is set to ``basic`` or ``token``
+        * ``remote_password`` is required if ``autehentication_type`` is set to ``basic``
+        * ``remote_token`` is required if ``autehentication_type`` is set to ``token``
+        * ``remote_auth_key`` is required if ``autehentication_type`` is set to ``key``
+
+-------
+Example
+-------
+^^^^^^^
+Request
+^^^^^^^
+
+``POST .../api/1/services/api/1/repo/add-remote.json``
+
+.. code-block:: json
+
+  {
+    "site_id" : "mysite",
+    "remote_name" : "myremote",
+    "remote_url" : "https://github.com/craftercms/remoterepo.git",
+    "authentication_type" : "basic",
+    "remote_username" : "joe.bloggs",
+    "remote_password" : "SuperSecret$$587"
+  }
+
+.. code-block:: json
+
+  {
+    "site_id" : "mysite",
+    "remote_name" : "myremote",
+    "remote_url" : "https://github.com/craftercms/remoterepo.git",
+    "authentication_type" : "token",
+    "remote_username" : "joe.bloggs",
+    "remote_token" : "SuperSecretToken"
+  }
+
+.. code-block:: json
+
+  {
+    "site_id" : "mysite",
+    "remote_name" : "myremote",
+    "remote_url" : "https://github.com/craftercms/remoterepo.git",
+    "authentication_type" : "key",
+    "remote_auth_key" : "SuperSecretKey"
+  }
+
+^^^^^^^^
+Response
+^^^^^^^^
+
+``Status 200 OK``
+
+.. code-block:: json
+
+  null
+
+---------
+Responses
+---------
+
++---------+-------------------------------------------+----------------------------------------------------------------+
+|| Status || Location                                 || Response Body                                                 |
++=========+===========================================+================================================================+
+|| 200    ||                                          || See example above.                                            |
```

Comment:
Replace 200 with 201. We will skip the location tag because we don't have a "getter" at this time.